gdk/gdk.go

```diff
@@ -1704,11 +1680,8 @@ func (v *Pixbuf) Flip(horizontal bool) (*Pixbuf, error) {
 		return nil, nilPtrErr
 	}
 
-	obj := &glib.Object{glib.ToGObject(unsafe.Pointer(c))}
-	p := &Pixbuf{obj}
-	//obj.Ref()
-	runtime.SetFinalizer(p, func(_ interface{}) { obj.Unref() })
-	return p, nil
+	obj := glib.Take(unsafe.Pointer(c))
```

Comment:
gdk_pixbuf_flip is "transfer full", no calling `Ref()`, just ensure `Unref()` is called.

Reply:
done